sixtyfps_runtime/corelib/items.rs

```diff
@@ -57,6 +57,22 @@ type ItemRendererRef<'a> = &'a mut dyn crate::item_rendering::ItemRenderer;
 pub type VoidArg = ();
 type KeyEventArg = (KeyEvent,);
 
+#[macro_export]
+macro_rules! declare_item_vtable {
+    (fn $getter:ident() -> $item_vtable_ty:ident for $item_ty:ty) => {
+        ItemVTable_static! {
+            /// The VTable for `$ty`
+            #[no_mangle]
+            pub static $item_vtable_ty for $item_ty
+        }
+        #[no_mangle]
+        pub extern "C" fn $getter() -> *const ItemVTable {
```

Comment:
Should we guard that against the #[cfg(feature="ffi")] ?  Also maybe #[cfg(platform="windows")]

Reply:
Yes! Thanks. Will do.

Reply:
Done in 5d6c8e7ee1b4f49076f145ad40354978f2ecaaa2